desci-contracts/Dockerfile

```
@@ -1,11 +1,12 @@
-FROM node:16.16.0
+FROM node:16.20.0-bookworm
 WORKDIR /app
 
 VOLUME /data
 VOLUME /app/.openzeppelin
 
 COPY ["package.json", "yarn.lock*", "./"]
 RUN yarn --production false
+RUN ldd --version | head -n1
```

I'd personally expand this in an `echo` call to make it really clear this is just a log line:
```suggestion
RUN echo "Checking libc version: $(ldd --version | head -n1)"
```